pkgs/tools/misc/tere/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+let
+  commonMeta = {
+    name = "tere";
+    version = "1.1.0";
+  };
+  tereSrc = fetchFromGitHub {
+    owner = "mgunyho";
+    repo = commonMeta.name;
+    rev = "v${commonMeta.version}";
+    sha256 = "BD7onBkFyo/JAw/neqL9N9nBYSxoMrmaG6egeznV9Xs=";
+  };
+in
+rustPlatform.buildRustPackage rec
+  {
+    pname = commonMeta.name;
+    version = commonMeta.version;
+
+    src = tereSrc;
+    cargoSha256 = "gAq9ULQ2YFPmn4IsHaYrC0L7NqbPUWqXSb45ZjlMXEs=";
+
+    # This test confirmed not working.
+    # https://github.com/mgunyho/tere/issues/44
+    cargoPatches = [ ./brokentest.patch ];
+
+    meta = with lib; {
+      description = "A faster alternative to cd + ls";
+      homepage = "https://github.com/mgunyho/tere";
+      license = licenses.eupl12;
+      maintainers = [ maintainers.ProducerMatt ];
+      platforms = platforms.all;
```

Comment:
```suggestion
```
set by buildRustPackage

Reply:
I like your suggestions. Should I rebase them into the original commit or add them as a separate commit?